package.json

```diff
@@ -108,7 +108,7 @@
     "@eslint/plugin-kit": "^0.2.5",
     "@humanfs/node": "^0.16.6",
     "@humanwhocodes/module-importer": "^1.0.1",
-    "@humanwhocodes/retry": "^0.4.1",
+    "@humanwhocodes/retry": "git+https://github.com/humanwhocodes/retry#pull/44/head",
```

Comment:
This change should be reverted after the dependency has been upgraded to a version that includes the fix humanwhocodes/retry#44. If the dependency's versions is upgraded in this PR, the PR's tag should be changed to "fix:".